core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -24,11 +24,14 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
```

Comment:
Please delete the unused import and reformat code

Reply:
Unused import removed